frontend/src/scalars/ui/Config.vue

```diff
@@ -1,26 +1,26 @@
 <template>
```

Comment:
Is there a particular reason for the renaming?

Reply:
Looks like Vue coding standard Component name starts with capital